db/migrate/20201004160728_add_order_to_navigation_links.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrderToNavigationLinks < ActiveRecord::Migration[6.0]
+  def change
+    add_column :navigation_links, :order, :integer
```

Comment:
Added a column called order that will allow the admin to save the order in which he'd like to display the nav items

Reply:
**question (non-blocking):** Can this column be null? If not, you probably should add the constraint. If yes, maybe we should extend the `order` statement in some way (e.g. `order(order: asc, name: :asc)`) to get an explicit consistent ordering? We could potentially also use a proc default to automatically populate this column, e.g.

```ruby
 default: -> { NavigationLink.maximum(:position).next 
```

There should always be few enough rows in this table that `maximum` doesn't become a bottleneck.